tests/Constraints/DefaultPropertiesTest.php

```diff
@@ -149,6 +149,16 @@ public function getValidTests()
                 '{"items":[{"default":null}]}',
                 '[null]'
             ),
+            array(// #21 items might be an array of schema (instead of an array of schema)
+                '[{}]',
+                '{"items":{"properties":{"propertyOne":{"default":"valueOne"}},"required":[]}}',
+                '[{"propertyOne":"valueOne"}]'
+            ),
+            array(// #22 if items is not an array, it does not create new item
+                '[]',
+                '{"items":{"properties":{"propertyOne":{"default":"valueOne"}},"required":[]}}',
```

Comment:
they were not, it's just how I found out about this bug and I didn't realized I could remove it from the tests